internal/apiclient/dashboard.go

```diff
@@ -129,3 +144,29 @@ func (c *DashboardAPIClient) AddRun(ctx *config.RunContext, out output.Root) (Ad
 	}
 	return response, nil
 }
+
+func (c *DashboardAPIClient) QueryOrgSettings() (QueryOrgSettingsResponse, error) {
+	response := QueryOrgSettingsResponse{}
+
+	q := `
+		query OrgSettings {
+        	orgSettings {
+            	cloudEnabled
+        	}
+    	}
+	`
+	results, err := c.doQueries([]GraphQLQuery{{q, map[string]interface{}{}}})
+	if err != nil {
+		return response, err
+	}
+
+	if len(results) > 0 {
+
+		if results[0].Get("errors").Exists() {
+			return response, errors.New(results[0].Get("errors").String())
```

Comment:
again let's wrap this error so it's easier to debug, e.g. `fmt.Errof("query failed when requesting org settings, received graphql error: %s", results[0].Get("errors").String())`